packages/interface/src/api/club/endpoint/apiClb004.ts

```diff
@@ -0,0 +1,51 @@
+import { HttpStatusCode } from "axios";
+import { z } from "zod";
+
+/**
+ * @version v0.1
+ * @description 동아리의 기본 정보를 가져옵니다
+ */
+
+const url = () => `/student/clubs/club/{club_id}/brief`;
+const method = "GET";
+
+const requestParam = z.object({
+  clubId: z.number().int(),
```

Comment:
```suggestion
  clubId: z.number().int().min(1),
```
❗️ id 는 전부 min(1) 걸어주기로 했어요!